doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -454,7 +454,9 @@ Bluetooth libraries and services
 Bootloader libraries
 --------------------
 
-|no_changes_yet_note|
+* :ref:`lib_dfu_target` library:
+
+   * Removed :c:func:`dfu_ctx_mcuboot_set_b1_file` (converted to a util function in :ref:`lib_fota_download`)
```

Comment:
Instead, you could say:
* Moved :c:func:`dfu_ctx_mcuboot_set_b1_file` to :ref:lib_fota_download` and renamed to...